package.json

```diff
@@ -4,7 +4,9 @@
   "description": "An HTTP/1.1 client, written from scratch for Node.js",
   "main": "index.js",
   "scripts": {
-    "test": "standard | snazzy && tap test/*.js --100 --no-ts --no-jsx --no-flow --no-esm"
+    "lint": "standard | snazzy",
+    "test": "tap test/*.js --no-coverage",
+    "coverage": "standard | snazzy && tap test/*.js --100 --no-ts --no-jsx --no-flow --no-esm"
```

Comment:
please move the `--no-ts --to-jsx --no-flow --no-esm` to a `.taprc` file. See https://github.com/fastify/fastify/blob/master/.taprc

Reply:
I guess coverage: true is the same as --100?